src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -146,23 +146,23 @@ function formReadReceiptsTooltip(
   const displayRecipientType = recipientType === "guardian" ? "parent" : recipientType;
   const recipientString = readBy.length === 1 ? displayRecipientType : `${displayRecipientType}s`;
   return (
-    <FlexBox className={cssClass("readReceipts--container")} alignItems="center" justify="end">
-      <div onMouseOver={onReadReceiptsHover}>
-        <Tooltip
-          className={cssClass("readReceipts--tooltip")}
-          content={readReceiptString}
-          placement={"top"}
-          textAlign={"left"}
-        >
-          <FlexBox>
+    <FlexBox alignItems="center" justify="end">
+      <Tooltip
+        className={cssClass("readReceipts--tooltip")}
+        content={readReceiptString}
+        placement={"top"}
+        textAlign={"left"}
+      >
+        <div onMouseEnter={onReadReceiptsHover}>
+          <FlexBox className={cssClass("readReceipts--container")}>
```

Comment:
I moved the container tag to here bc there was a bug where the tooltip would turn blue upon hover across the entire width of the AnnouncementBubble component rather than just the tooltip itself.